cypress/integration/banner.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { banner } from '../support/config'
+
+describe('Banner', () => {
+  beforeEach(() => {
+    cy.visit('/')
+    expect(localStorage.getItem('bannerTimeStamp')).to.be.null
+  })
+
+  it('text correct', () => {
```

Comment:
If cypress handles the test cases like other frameworks, then the content should be written like if you want to complete a sentence. In this case it would be "shows the correct alert banner text"

Reply:
I changed some of the descriptions